components/compliance-service/ingest/pipeline/publisher/node_manager_publisher.go

```diff
@@ -0,0 +1,122 @@
+package publisher
+
+import (
+	"context"
+	"time"
+
+	"github.com/chef/automate/components/compliance-service/ingest/events/compliance"
+	"github.com/chef/automate/components/compliance-service/ingest/pipeline/message"
+	"github.com/chef/automate/components/nodemanager-service/api/manager"
+	"github.com/chef/automate/components/nodemanager-service/api/nodes"
+	"github.com/golang/protobuf/ptypes"
+	tspb "github.com/golang/protobuf/ptypes/timestamp"
+	log "github.com/sirupsen/logrus"
+)
+
+func BuildNodeManagerPublisher(nodeManagerClient manager.NodeManagerServiceClient) message.CompliancePipe {
+	return func(in <-chan message.Compliance) <-chan message.Compliance {
+		return nodeManagerPublisher(in, nodeManagerClient)
+	}
+}
+
+func nodeManagerPublisher(in <-chan message.Compliance, nodeManagerClient manager.NodeManagerServiceClient) <-chan message.Compliance {
+	ctx := context.Background()
+	maxNumberOfBundledMsgs := 100
+	out := make(chan message.Compliance, maxNumberOfBundledMsgs)
+	go func() {
+		for msg := range in {
+			// send to node manager from here.
+			log.Infof("send info about node %s to node manager", msg.Report.NodeName)
+
+			nodeMetadata, err := gatherInfoForNode(msg)
```

Comment:
had to move the call to process node to the compliance ingestion publisher so i could access projects as well as node status